.github/workflows/deployments.yml

```diff
@@ -258,6 +258,43 @@ jobs:
         env:
           NETLIFY_AUTH_TOKEN: ${{ secrets.TEST_NETLIFY_TOKEN }}
```

Comment:
Maybe we should also add it here for consistency?

```suggestion
          NETLIFY_AUTH_TOKEN: ${{ secrets.TEST_NETLIFY_TOKEN }}
          USE_NETLIFY_EDGE: false
```

Reply:
We don't want it here because it's literally only for CI for Netlify Edge. Just the presence of the environment variable is all we care about which is why I didn't include it for the Netlify Functions deploy.

Reply:
Our integration tests are also rewritten to be explicit about both cases, so that's why I think we should do it here as well

Reply:
env vars are strings, so you'd need to change the comparison logic if you do this, because `"false"` is truthy.

Reply:
It wouldn't be a problem to do the following check imo

```ts
if(USE_NETLIFY_EDGE === "true") {
  // ...
}
```